API/users/models.py

```diff
@@ -1,2 +1,46 @@
+from django.db import models
+from django.contrib.auth.models import User
 
 # Create your models here.
+
+class Account(models.Model):
+    # account_set added implicitly
+    name = models.CharField(max_length=64)
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    def __str__(self):
+        return self.name
```

Comment:
idk whether we need `__str__` on all classes, but if we need, maybe let's make the result more detailed?
sth like this
```suggestion
        return f"Account(name: '{self.name}', user_id:'{self.user_id}')"
```
?

Reply:
I think we can leave `__str__` as it is. More detailed info can be returned by `__repr__`. That being said, probably we won't need any of those lol